src/alchemlyb/tests/parsing/test_namd.py

```diff
@@ -5,6 +5,8 @@
 
 from alchemlyb.parsing.namd import extract_u_nk
 from alchemtest.namd import load_tyr2ala
+from alchemtest.namd import load_idws
```

Comment:
This dataset only contains a single file; to test the functionality of parsing multiple files we should have an appropriate data set (or if it makes sense), double the existing data to fake it (maybe shuffle it?)